onnxruntime/core/providers/cpu/cpu_execution_provider.cc

```diff
@@ -24,13 +24,41 @@ struct KernelRegistryAndStatus {
 }  // namespace
 
 namespace onnxruntime {
+CPUExecutionProvider::CPUExecutionProvider(const CPUExecutionProviderInfo& info, bool delay_allocator_registration)
+    : IExecutionProvider{onnxruntime::kCpuExecutionProvider}, info_{info} {
+  if (!delay_allocator_registration) {
+    AllocatorManager mgr;
```

Comment:
This is strictly not needed; it's just here to reuse the RegisterAllocator method.